<!DOCTYPE HTML>
<!--
	Linear by TEMPLATED
    templated.co @templatedco
    Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
-->
<html>
	<head>
		<title>BioDS</title>
		<link rel="shortcut icon" type="image/jpg" href="images/biomeds.png"/>
		<meta http-equiv="content-type" content="text/html; charset=utf-8" />
		<meta name="description" content="" />
		<meta name="keywords" content="" />
		<link href='http://fonts.googleapis.com/css?family=Roboto:400,100,300,700,500,900' rel='stylesheet' type='text/css'>
		<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
		<script src="js/skel.min.js"></script>
		<script src="js/skel-panels.min.js"></script>
		<script src="js/init.js"></script>
		<noscript>
			<link rel="stylesheet" href="css/skel-noscript.css" />
			<link rel="stylesheet" href="css/style.css" />
			<link rel="stylesheet" href="css/style-desktop.css" />
		</noscript>
	</head>
	<body>

	<!-- Header -->
		<div id="header">
			<div id="nav-wrapper"> 
				<!-- Nav -->
				<nav id="nav">
					<ul>
						<li><a href="index.html">Homepage</a></li>
						<li><a href="projects.html">Projects</a></li>
						<li><a href="members.html">People</a></li>
						<li><a href="proposals.html">MSc Proposals</a></li>
						<li><a href="https://github.com/bio-datascience">Tools</a></li>
						<li><a href="https://scholar.google.com/citations?hl=en&user=zplWERYAAAAJ&view_op=list_works&sortby=pubdate">Publications</a></li>
						<li><a href="funding.html">Funding</a></li>

					</ul>
				</nav>
			</div>
			<div class="container"> 
				
				<!-- Logo -->
				<div id="logo">
					<h1><a href="#">Projects</a></h1>
					<span class="tag"></span>
				</div>
			</div>
		</div>
	<!-- Header --> 

	<!-- Featured -->
	<div id="featured">
		<div class="container">
			<header>
				<h2></h2>
			</header>
			<p>Our group is involved in multiple projects, ranging from the study of microbial communities to the dissection of epigenetic datasets.
				<br><br>
				A few of our main projects are listed below. 
				<br><br>
				We are always open to collaborations</p>
		</div>
	</div>

	<!-- Main -->
		<div id="main">
			<div id="content" class="container">
				<section>
					<header>
						<h2></h2>
					</header>
				</section>
				<div class="row">
					<div class="span3" alignment="center">
						<a target="_blank" href="https://sccoda.readthedocs.io" class="image full"><img src="images/scCODA_concept_fig.png" alt=""></a>
					</div>
					<div class="span3">
						<span class="byline"><strong><a target="_blank" href="https://sccoda.readthedocs.io/">scCODA </a></strong></span>
						<h6 align="justify">
							Compositional changes of cell types are main drivers of biological processes. 
							However, their detection through single-cell experiments is difficult due to the compositionality of the data and 
							low sample sizes. <br>
							scCODA, a Bayesian model addressing these issues, enables the study of complex cell type effects in disease, 
							and other stimuli. scCODA demonstrates excellent detection performance and could identify experimentally verified 
							cell type changes that were missed in original analyses. 
							The method was developed by <strong><a href="members.html">Johannes</a></strong> in collaboration with the groups of 
							<strong><a href="https://www.helmholtz-muenchen.de/icb/research/groups/schubert-lab/overview/index.html" target="_blank">Benjamin Schubert</a></strong> and 
							<strong><a href="https://www.helmholtz-muenchen.de/icb/research/groups/theis-lab/overview/index.html" target="_blank">Fabian Theis</a></strong>. <br>
							You can read more about scCODA in <strong><a href="https://www.nature.com/articles/s41467-021-27150-6" target="_blank">Nature Communications</a></strong> and <strong><a href="https://sccoda.readthedocs.io/" target="_blank">readthedocs</a></strong>.
						</h6>.
					</div>
				</div>

				<div class="row">
					<hr style="width:100%;text-align:left;margin-left:0">
				</div>

				<div class="row">
					<div class="span3" alignment="center">
						<a target="_blank" href="https://bayresq.net/en/projekte-stressregnet-en/" class="image full"><img src="images/stressregnet_small.png" alt=""></a>
					</div>
					<div class="span3">
						<span class="byline"><strong><a target="_blank" href="https://bayresq.net/en/projekte-stressregnet-en/">StressRegNet</a></strong></span>
						<h6 align="justify">
							In this project, we aim to identify stressors and bacterial regulatory pathways that control host adaptation and antibiotic sensitivity. By using a high-throughput chemical genomics approach, we will explore chemical stimuli from the host environment that induce stress response pathways in Salmonella and Campylobacter, and elucidate the underlying molecular crosstalk between sensory pathways of these microbes. 
							This project is part of the bavarian <strong><a target="_blank" href="https://bayresq.net/">Bayresq</a></strong> research consortium, and is done in close colaboration with members of
							<strong><a target="_blank" href="http://www.imib-wuerzburg.de/research/sharma/research/">Dr. Cynthia Sharma's</a></strong> and <strong><a target="_blank" href="https://www.brochadolab.com/">Dr. Ana Rita Brochado's</a></strong> labs. For more info on the role of our group in this project you can contact <strong><a href="members.html">Roberto</a></strong>
						</h6>.
					</div>
				</div>
				
				<div class="row">
					<hr style="width:100%;text-align:left;margin-left:0">
				</div>

				<div class="row">
					<div class="span3" alignment="center">
						<a target="_blank" href="https://www.helmholtz-muenchen.de/ife/research/robert-schneider-chromatin-dynamics-and-epigenetics/research/index.html" class="image full"><img src="images/combinatorial_small.png" alt=""></a>
					</div>
					<div class="span3">
						<span class="byline"><strong><a target="_blank" href="https://www.helmholtz-muenchen.de/ife/research/robert-schneider-chromatin-dynamics-and-epigenetics/research/index.html/">Combinatorial Epigenetics</a></strong></span>
						<h6 align="justify">
							In this project, we combine chemical biology, biochemical, and proteomic approaches with cell biology and computational analyses to unravel how epigenetic effector molecules can read DNA and histone modification patterns and how they recognise different chromatin modification states. Our goal is to decipher the “epigenetic code” by identifying epigenetic reader molecules that can integrate information from multiple chromatin modifications and to understand how these factors operate at the molecular level both in healthy and pathological conditions. 
							<br>
							This project is done in close collaboration with <a href="https://www.helmholtz-muenchen.de/ife/about-us/people/staff-detail/ma/5831/Dr-Bartke/index.html" target="_blank">Dr. Till Bartke</a> from the Institute of Functional Epigenetics at Helmholtz Zentrum München. In our group <strong><a href="members.html">Mara</a></strong> is responsible for this project.

						</h6>.
					</div>
				</div>

				<div class="row">
					<hr style="width:100%;text-align:left;margin-left:0">
				</div>

				<div class="row">
					<div class="span3" alignment="center">
						<a target="_blank" href="https://github.com/bio-datascience/NetCoMi" class="image full"><img src="images/NetCoMi_workflow_small.png" alt="" width="550" height="370"></a>
					</div>
					<div class="span3">
						<span class="byline"><strong><a target="_blank" href="https://github.com/bio-datascience/NetCoMi">NetCoMi (Network Construction and Comparison for Microbiome Data)</a></strong></span>
						<h6 align="justify">
							NetCoMi is an R package that provides an extensive toolbox for constructing, analyzing, and comparing networks for microbial compositional data. 
							It also offers the ability to quantify network differences and thus enables insights into whether single taxa, groups of taxa, or the overall network structure change between groups. 
							The package was developed by <strong><a href="members.html">Stefanie</a></strong> as part of her Ph.D. thesis, in collaboration with her supervisors and the group of 
							<strong><a href="https://www.helmholtz-munich.de/en/iap" target="_blank">Prof.Dr. Erika von Mutius</a></strong> at Helmholtz Zentrum München.
							The manuscript describing the workflow and theory behind the package is published 
							in <strong><a href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbaa290/6017455" target="_blank">Briefings in Bioinformatics</a></strong>.
						</h6>.
					</div>
				</div>
				
				<div class="row">
					<div class="span3" alignment="center">
						<a target="_blank" href="https://github.com/bio-datascience/tascCODA" class="image full"><img src="images/group_website_tascCODA.png" alt=""></a>
					</div>
					<div class="span3">
						<span class="byline"><strong><a target="_blank" href="https://github.com/bio-datascience/tascCODA">tascCODA </a></strong></span>
						<h6 align="justify">
							Microbial or cell type compositions from high-throughput sequencing experiments are often ordered by a tree structure, such as taxonomic or cell lineage trees.
							The <strong><a href="https://www.frontiersin.org/articles/10.3389/fgene.2021.766405/full" target="_blank">tascCODA</a></strong> method, which was developed by <strong><a href="members.html">Johannes</a></strong>,
							utilizes this structure to dynamically infer compositional changes on subgroups of the composition. <br>
							Compared with other methods, tascCODA has very good detection accuracy and gives a more parsimonious picture of the compositional shifts in the data.
							The method was published in <strong><a href="https://www.frontiersin.org/articles/10.3389/fgene.2021.766405/full" target="_blank">Frontiers in Genetics</a></strong>, and the Python implementation can be found on our <strong><a href="https://github.com/bio-datascience/tascCODA" target="_blank">github page</a></strong>.
						</h6>.
					</div>
				</div>

			</div>
		</div>
	<!-- /Main -->

	<!-- Tweet -->
		<div id="tweet">
			<div class="container">
				<section>
					<blockquote>&ldquo;Biomedical Statistics and Data Science&rdquo;</blockquote>
				</section>
			</div>
		</div>
	<!-- /Tweet -->

	<!-- Footer -->
	<div id="footer">
		<div class="container">
			<section>
				<header>
					<h2>Get in touch</h2>
					<span class="byline">You can find us at</span>
				</header>
				<ul class="contact">
					<li><a target="_blank" href="https://twitter.com/microbionaut" class="fa fa-twitter"><span>Twitter</span></a></li>
					<li target="_blank" class="active"><a href="https://github.com/bio-datascience" class="fa fa-github"><span>Facebook</span></a></li>
					<li><a href="mailto:christian.mueller@helmholtz-muenchen.de?Subject=Hello%20User" class="fa fa-envelope"><span>Pinterest</span></a></li>
				</ul>
			</section>
		</div>
	</div>
	<!-- /Footer -->

	<!-- Copyright -->
		<div id="copyright">
			<div class="container">
				Design: <a href="http://templated.co">TEMPLATED</a> Images: <a href="http://unsplash.com">Unsplash</a> (<a href="http://unsplash.com/cc0">CC0</a>)
			</div>
		</div>


	</body>
</html>
